### PR TITLE
Taxa clustering update

### PR DESCRIPTION
**Pull request by Basil-Courbayre-Dussau**
This PR is supposed to correct what went wrong in PR #295.

### CODE REVIEW

**Comment by antagomir** [comment]
Status of this PR?

**Comment by BananaCancer** [comment]
Sorry for the delay, I fixed the merging problems and will now try to fix the bluster dependency: bluster old version is used in the check so the check fails. 
For now I've put an install of the github version of bluster in the check but that may not be ideal.
The rest is ready, tell me if you have any final addition you want me to make.
I've tried adding bluster in the Suggests of DESCRIPTION but that changed nothing.
@antagomir @TuomasBorman

File: 11_taxonomic_information.Rmd [hunk 1]
```diff
@@ -166,7 +168,10 @@ assay(altExp(tse, "Family"), "counts")[1:5,1:7]
 `altExpNames` now consists of `Family` level data. This can be extended to use 
 any taxonomic level listed in `mia::taxonomyRanks(tse)`.
 
-Rare taxa can also be aggregated into a single group "Other" instead of filtering them out. A suitable function for this is `agglomerateByPrevance`. The number of rare taxa is higher on the species level, which causes the need for data agglomeration by prevalence.
+Rare taxa can also be aggregated into a single group "Other" instead of 
+filtering them out. A suitable function for this is `agglomerateByPrevance`.
```
Comment (antagomir) [inline 11_taxonomic_information.Rmd:172]: agglomerateByPrevance -> agglomerateByPrevalence

File: 11_taxonomic_information.Rmd [hunk 2]
```diff
@@ -185,12 +278,15 @@ such as the centered log-ratio transformation (clr).
...
+when MARGIN = 'features'.
 
 For a complete list of available transformations and parameters, see function 
 [help](https://microbiome.github.io/mia/reference/transformAssay.html).
 
 ```{r}
+tse <- tseGlobalPatterns
 tse <- transformAssay(tse, assay.type = "counts", method = "relabundance", pseudocount = 1)
 tse <- transformAssay(x = tse, assay.type = "relabundance", method = "clr", 
                         pseudocount = 1, name = "clr_transformation")
```
Comment (antagomir) [inline 11_taxonomic_information.Rmd:292]: "clr" would be enough for a good assay name ("clr_transformation" is a bit long to write when using these)

File: 11_taxonomic_information.Rmd [hunk 3]
```diff
@@ -175,6 +180,94 @@ altExp(tse, "Species_byPrevalence")
...
+```{r taxa_clustering_histogram}
+library(ggplot2)
+library(patchwork) # TO arrange several plots as a grid
+plot1 <- ggplot(as.data.frame(rowData(tse)), aes(x = clusters_euclidean)) + 
+    geom_bar() + 
+    labs(title = "CAG size distribution (Euclidean distance)",
+         x = "Clusters", y = "Feature count (n)")
+plot2 <- ggplot(as.data.frame(rowData(tse)), aes(x = clusters_kendall)) + 
+    geom_bar() + 
+    labs(title = "CAG size distribution (1 - tau distance)",
```
Comment (antagomir) [inline 11_taxonomic_information.Rmd:257]: "(1 - tau distance)" -> "(1 - tau dissimilarity)" or just "(1-tau)"  - I suggest the latter

**Review by antagomir — COMMENTED** [review]
Very nice. Just minor observations, then ready from my perspective.

**Review by TuomasBorman — APPROVED** [review]

**Comment by antagomir** [comment]
to merge?

**Pull request merged**